source/_integrations/msteams.markdown

```diff
@@ -0,0 +1,54 @@
+---
+title: "Microsoft Teams"
+description: "Instructions on how to send notification to a Microsoft Teams channel."
+logo: msteams.png
+ha_category:
+  - Notifications
+ha_release: 0.101
+---
+
+The `Microsoft Teams` platform allows you to send notifications from Home Assistant to a team channel in [Microsoft Teams](https://products.office.com/en-us/microsoft-teams/group-chat-software).
+
+## Setup
+
+To send notification to teams you need add the Incoming Webhook app to your team channel. When the app is added you will receive a webhook url that needs to be added to your `configuration.yaml`.
+
+## Configuration
+
+To add the Microsoft Teams platform to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+notify:
+  - platform: msteams
+    name: myteam
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.